server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java

```diff
@@ -156,97 +156,97 @@
       .defaultValue(10000000L)
       .withDescription("Threshold for event size");
 
-  public static final ConfigOption<Double> RSS_CLEANUP_THRESHOLD = ConfigOptions
+  public static final ConfigOption<Double> CLEANUP_THRESHOLD = ConfigOptions
```

Comment:
why remove all RSS prefix in this PR, it's not related and change too many files

Reply:
address macduan's comments. keep consistent with other configuration option.